moderation-commands/clean.md

```diff
@@ -25,19 +25,19 @@ For more information on editing permission requirements for specific users/roles
 
 ### Options
 
-* `amount`: The amount of messages you want to clean (The default/maximum amount is 100 messages)
+* `amount`: The amount of messages you want to clean (The default/maximum amount is 100)
 * `user`: The user you want to do delete the messages of
 * `starts-with`: Delete messages that start with this value
 * `ends-with`: Delete messages that end with this value
 * `contains`: Delete messages that contain this value
-* `bots`: Whether or not to delete messages sent by bots
+* `bots`: Whether or not to delete messages sent by bots. If this option is left blank, the bot will delete these messages, but if left as true, the bot will only delete messages by bots, otherwise it will only delete messages not sent by bots
```

Comment:
left as true? this sentence is generally very convoluted and needs rewording

and the system options works the same way